src/Microsoft.ML.Data/DataView/ChainedRowToRowMapper.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.ML.Runtime.Internal.Utilities;
+
+namespace Microsoft.ML.Runtime.Data
+{
+    /// <summary>
+    /// A row-to-row mapper that is the result of a chained application of multiple mappers.
+    /// </summary>
+    public sealed class ChainedRowToRowMapper : IRowToRowMapper
```

Comment:
>Chained [](start = 24, length = 7)

Can we rename it either to `CompositeRowToRowMapper` (as this is similar to `CompositeDataLoader`) or `RotToRowMapperChain` (akin to `TransformerChain` and `EstimatorChain`) ?

Reply:
Let me do the composite thing.

---
In reply to: [219555464](https://github.com/dotnet/machinelearning/pull/973#discussion_r219555464) [](ancestors = 219555464)